groovy/material/ThermodynamicsMaterials.groovy

```diff
@@ -0,0 +1,317 @@
+package material;
+
+import static material.SuSyMaterials.*;
+
+import gregtech.api.unification.material.Material;
+import gregtech.api.GregTechAPI;
+import gregtech.api.fluids.fluidType.FluidTypes;
+import gregtech.api.unification.material.properties.*
+
+import static gregtech.api.unification.material.info.MaterialIconSet.*;
+import static gregtech.api.unification.material.info.MaterialFlags.*;
+import static gregtech.api.unification.material.Materials.*;
+
+
+class ThermodynamicsMaterials {
```

Comment:
This class has a lot of parameters called `materialName` of type `Material`. This name can be confusing, so I suggest renaming it to just `material`, or something else which is more accurately descriptive of its use.